### ISSUE TITLE
Ruby install fails - no GCC

### ISSUE THREAD

**Issue by das3in** [thoughtbot/laptop#536]
Looks like GCC no longer ships with XCode - needs to be installed separately

```
If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

Cloning into '/Users/user/.asdf/plugins/ruby'...
Cloning into '/Users/user/.asdf/plugins/nodejs'...

Installing latest Ruby...
Downloading ruby-build...
Cloning into 'ruby-build-source'...
Note: checking out 'v20180424'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at d7fa3ad bump version to v20180424
No versions installed

ERROR: This package must be compiled with GCC, but ruby-build couldn't
find a suitable `gcc` executable on your system. Please install GCC
and try again.

DETAILS: Apple no longer includes the official GCC compiler with Xcode
as of version 4.2. Instead, the `gcc` executable is a symlink to
`llvm-gcc`, a modified version of GCC which outputs LLVM bytecode.

For most programs the `llvm-gcc` compiler works fine. However,
versions of Ruby older than 1.9.3-p125 are incompatible with
`llvm-gcc`. To build older versions of Ruby you must have the official
GCC compiler installed on your system.

TO FIX THE PROBLEM: Install Homebrew's apple-gcc42 package with this
command: brew tap homebrew/dupes ; brew install apple-gcc42

You will need to install the official GCC compiler to build older
versions of Ruby even if you have installed Apple's Command Line Tools
for Xcode package. The Command Line Tools for Xcode package only
includes `llvm-gcc`.

BUILD FAILED (OS X 10.13.3 using ruby-build 20180424)

failed
```

**Comment by das3in** [comment]
Looks like this was discussed a while ago in #25 - conclusion was to leave it out of the laptop script

**Comment by purinkle** [comment]
This is the same as #533

**Comment by composerinteralia** [comment]
#534 fixed #533, so I am going to close this. Thanks!
